### DISCUSSION TITLE
A/B flag to deactivate speed up when tap and hold on video

### DISCUSSION THREAD

**Discussion by theoccurrence** [PoomSmart/YTABConfig#94] [General]
Title pretty much says it all. Most annoying feature they added in a looong time. Usually I don't mind changes like layout changes etc., but this feature goes out of it's was to annoy me every single time.

**Comment by ahamza61** [comment]
This issue is still a thing. Please fix.

**Reply by Scarborrower** [reply]
`YTColdConfig.cxClientEnableVariableSpeedPlayback`

**Reply by sirlupusdev** [reply]
This didn't fix it for me when using 18.45.2, most likely because the code got removed, but it obviously may help others, thank you!

**Comment by PoomSmart** [comment]
`YTColdConfig.speedMasterArm2FastForwardWithoutSeekBySliding` set to false
